packages/by-name/microsoft/genpolicy/package.nix

```diff
@@ -71,6 +71,10 @@ rustPlatform.buildRustPackage rec {
       # This avoids printing the entire annotation on log level debug, which resulted in errors of the logtranslator.go
       # TODO(jmxnzo): remove when https://github.com/kata-containers/kata-containers/pull/10647 is picked up by microsoft/kata-containers fork
       ./0009-genpolicy-do-not-log-policy-annotation-in-debug.patch
+      # Patches the RootfsPropagation check in allow_create_container_input to allow setting up volumes. It reflects
```

Comment:
How exactly are RootfsPropagation and volume setup connected?

Reply:
Volume propagation allows creating a mount point in one container (== mount namespace) and accessing it in another. Setting the container's [volumeMount to `bidirectional` ](https://github.com/edgelesssys/contrast/blob/662af0c011bd68bb09b78f8bf96427d84250a0b1/internal/kuberesource/sets.go#L536) makes containerd add the `RootfsPropagation` to the pod. I'm not sure if it's strictly necessary - I'd have expected only the mount point to need propagation - but it should also not harm because the container rootfs is not mounted elsewhere.

That being said, I still owe an upstream issue and PR for this - will start with that once I have some cycles.